KMaina/Fast-Food-App
# #160292841 Page to Allow Users Order For Food

#### What does this PR do?
Allow a user to order for a meal from a list of meals
#### Description of Task to be completed?
The user should be able to

- Select a meal from the list provided.


#### How should this be manually tested?

- Clone the repo onto your local machine.

- Open a terminal and run `cd Fast-Food-App`

- run `git checkout -b ft-sign-in-page-160292841`
 
- run `git pull https://github.com/KMaina/Fast-Food-App.git ft-user-orders-page-160292841`

- In the folder where you cloned the repo, you should open the UI folder.
 
- Open the user-order.html to see it in your browser for review.

#### Any background context you want to provide?
This is simply the UI design, no functionality has been built in yet.
#### What are the relevant pivotal tracker stories?
#160292841
